config/db_tools.standalone.complete.sample.yaml

```diff
@@ -54,7 +54,7 @@
 # Connection name will be "default" when configured this way.
 connections: "pgsql://username:password@hostname:port?version=16.0&other_option=..."
 
-# Alternatively, you are allowed to have more than one connections, case
+# Alternatively, you are allowed to have more than one connection, case
```

Comment:
"More" than one, shouldn't be plural "connections" here ?

Reply:
Or that may be a frenchism, I'm not sure about english grammar here.

Reply:
I checked on Linguee and the singular seems correct.